inst/extdata/packages.R

```diff
@@ -0,0 +1,94 @@
+# Install repgen, production instance, prerequisite packages. Should be safe to
+# run repeatedly without additional intervention.
+
+# devtools is currently only required on a production instance to install gsplot
+# from GitHub (see below). There are plans to "promote" it to CRAN (or GRAN?),
+# but that hasn't happened yet. See 
+# https://usgs-cida.slack.com/archives/aqcu/p1484934743005621 if it still 
+# exists.
+pkgs <- c(
+  "devtools",
+  "dplyr",
+  "evaluate",
+  "htmlTable",
+  "jsonlite",
+  "lazyeval",
+  "knitr",
+  "lubridate",
+  "rmarkdown",
+  "yaml"
+)
+
+# Known production tiers, listed at
+# https://docs.google.com/document/d/1vBOTUPtIdeCTGe7Or2cQGMMOFS_hD545yZ1E6YBGizs/edit
+tiers <-
```

Comment:
If the desire is to have this parameterized, it could be a simple "includeDevPackages" boolean, and the caller would decide what to pass in. Versus the script code deciding based on where it's sitting.